scalding-core/src/main/scala/com/twitter/scalding/FlowState.scala

```diff
@@ -21,9 +21,12 @@ import java.util.WeakHashMap
 /**
  * Immutable state that we attach to the Flow using the FlowStateMap
  */
-case class FlowState(sourceMap: Map[String, Source] = Map.empty) {
+case class FlowState(sourceMap: Map[String, Source] = Map.empty, flowBoxes: Set[(String, String)] = Set()) {
```

Comment:
what about changing flowBoxes to `flowConfigUpdates` or something. It seems like it is not really to do with the serialization and could be used for any global flow config key/value updates.